### PR TITLE
streamline updating of country-specific cloud.yamls, fixes #2454

### LINKED ISSUE #2454: Streamline the generation of country-specific `cloud.yaml`s
Right now the process of generating and uploading 'cloud.cn.yaml' and 'cloud.default.yaml' is a multi-step process that requires a working checkout of the 'lantern_aws' project.  It should be possible to make it a one-command operation and only depend on 'too-few-secrets'.

### PR DESCRIPTION
**Pull request by aranhoide**

### CODE REVIEW

**Comment by aranhoide** [comment]
Note I didn't commit an updated masquerades.go etc., to remove git noise.  Feel free to do it once you're done with QA, or let me know when this looks good to you and I'll commit them before merging.

File: src/github.com/getlantern/flashlight/genconfig/uploadyaml.bash [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  echo $*
+  exit 1
+}
+
+if [ -z "$1" ]
+then
+    die "Need to specify version string(s), e.g., '$0 default cn'"
+fi
+
+for version
```
Comment (myleshorton) [inline src/github.com/getlantern/flashlight/genconfig/uploadyaml.bash]: Where is version defined?

Comment (aranhoide) [inline src/github.com/getlantern/flashlight/genconfig/uploadyaml.bash]: This is a shortcut for iterating over the arguments, so if you call `uploadyaml.bash cn default`, `version` takes the values `cn` and `default` in turn.

File: src/github.com/getlantern/flashlight/genconfig/generate-fallbacks-json.bash [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+function die() {
+  echo $*
+  exit 1
+}
+
+[ -n "$1" ] || die "Usage: '$0 country-code country-code' ...'    (e.g. '$0 nl jp')"
+[ -n "${production_cloudmaster_IP}" ] || die "You need to set the production_cloudmaster_IP environment variable."
+
+for locale
```
Comment (aranhoide) [inline src/github.com/getlantern/flashlight/genconfig/generate-fallbacks-json.bash]: This is the same shortcut as described below.  If you run `./generate-fallbacks-json.bash nl jp`, `locale` will take the values `nl` and `jp` in respective iterations of the loop.

File: src/github.com/getlantern/flashlight/genconfig/uploadyaml.bash (hunk 1 above)
Comment (myleshorton) [inline src/github.com/getlantern/flashlight/genconfig/uploadyaml.bash]: Wow, that's cool. Never knew that!

**Comment by aranhoide** [comment]
No more country-specific cloud.yamls!

**Pull request closed without merging**